cobra/cmd/license_lgpl.go

```diff
@@ -0,0 +1,188 @@
+package cmd
+
+func initLgpl() {
+	Licenses["lgpl"] = License{
+		Name:            "GNU Lesser General Public License",
+		PossibleMatches: []string{"lgpl", "lesser gpl", "gnu lgpl"},
```

Comment:
Sorry to bug you with all this BTW.... but I am maintainer a license scanner tool... so I know a tad too many things about licenses. And may be I care too much about accuracy there ;)